src/views/Restoration/RestorationBusinessName.vue

```diff
@@ -58,19 +62,22 @@ import { useStore } from '@/store/store'
 import { CommonMixin } from '@/mixins'
 import { RouteNames } from '@/enums'
 import { CorpTypeCd } from '@bcrs-shared-components/enums/'
+import {VDivider} from 'vuetify/lib'
```

Comment:
There's no need to import it Paul since it's a Vuetify component. You can remove this import (and below) and see the v-divider still shows.

Reply:
Fixed, thanks! Resolving.